webrender/src/debug_render.rs

```diff
@@ -260,29 +318,31 @@ impl DebugRenderer {
         self.add_line(p0.x, p1.y, color, p0.x, p0.y, color);
     }
 
-    pub fn render(
+    pub fn render<B: hal::Backend>(
         &mut self,
-        device: &mut Device,
+        device: &mut Device<B>,
         viewport_size: Option<DeviceUintSize>,
     ) {
         if let Some(viewport_size) = viewport_size {
             device.disable_depth();
             device.set_blend(true);
             device.set_blend_mode_premultiplied_alpha();
 
-            let projection = Transform3D::ortho(
+            let projection = create_projection(
                 0.0,
                 viewport_size.width as f32,
                 viewport_size.height as f32,
                 0.0,
-                ORTHO_NEAR_PLANE,
-                ORTHO_FAR_PLANE,
+                true,
             );
 
             // Triangles
             if !self.tri_vertices.is_empty() {
                 device.bind_program(&self.color_program);
-                device.set_uniforms(&self.color_program, &projection);
+                device.set_uniforms(
```

Comment:
I agree that it looks better in multiple line, but we should use the original syntax.